integration/container/create_test.go

```diff
@@ -225,6 +225,83 @@ func TestCreateWithCustomMaskedPaths(t *testing.T) {
 	}
 }
 
+func TestCreateWithCapabilities(t *testing.T) {
+	skip.If(t, testEnv.DaemonInfo.OSType != "linux")
+
+	defer setupTest(t)()
+	client := request.NewAPIClient(t)
+	ctx := context.Background()
+
+	testCases := []struct {
+		capabilities []string
+		expected     []string
+	}{
+		{
+			capabilities: []string{"CAP_NET_RAW", "CAP_SYS_CHROOT"},
+			expected:     []string{"CAP_NET_RAW", "CAP_SYS_CHROOT"},
+		},
+		{
+			capabilities: []string{"CAP_NET_RAW", "CAP_SYS_CHROOT"},
```

Comment:
This case is the same as the previous one?


We may need a case;

- without a `CAP_` prefix, to check that it produces an error (and check if it's the right type of error)
- with both `capabilities` and `capAdd` set (check for the conflicting options error)
- with both `capabilities` and `capDrop` set (check for the conflicting options error)
- with duplicate entries in the request (e.g. `[]string{"CAP_NET_RAW", "cap_net_raw"}`)
- _ideally_ we should think of testing agains an older version of the API, and verify the option is ignored

Reply:
These are now included to tests but notice that duplicates are fully valid case as Linux kernel will just ignore them.

Reply:
Yes, that's actually why I wanted to have a test; to check that we allow duplicates, and that we preserve them